### PR TITLE
componentize mvs changes, same as jes

### PR DESCRIPTION
**Pull request by NakulManchanda**
Signed-off-by: Nakul Manchanda <nakul.manchanda@ibm.com>

### CODE REVIEW

File: scripts/explorer-mvs-configure.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# Copyright IBM Corporation 2019
+################################################################################
+
+################################################################################
+# Variables required on shell:
+# - STATIC_DEF_CONFIG_DIR
+# - ROOT_DIR
+# - NODE_HOME
+
+. $ROOT_DIR/scripts/utils/validateNode.sh
```
Comment (stevenhorsman) [inline scripts/explorer-mvs-configure.sh]: This should be in component validate.sh, or the run-zowe.sh, not in component configure.sh

File: scripts/explorer-mvs-start.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# Variables required on shell:
+# - NODE_HOME
+# - ROOT_DIR
+# - ZOWE_EXPLORER_HOST
+# - GATEWAY_PORT
+# - MVS_EXPLORER_UI_PORT
+# - KEYSTORE_KEY
+# - KEYSTORE_CERTIFICATE
+
+. $ROOT_DIR/scripts/utils/validateNode.sh
```
Comment (stevenhorsman) [inline scripts/explorer-mvs-start.sh]: This should be in component validate.sh, or the run-zowe.sh, not in component configure.sh

File: scripts/explorer-mvs-validate.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# Copyright IBM Corporation 2019
+################################################################################
+
+################################################################################
+# Variables required on shell:
+# - STATIC_DEF_CONFIG_DIR
+# - ROOT_DIR
+# - NODE_HOME
+
+. $ROOT_DIR/scripts/utils/validateNode.sh
```
Comment (stevenhorsman) [inline scripts/explorer-mvs-validate.sh]: Can you rename this to validate-node please to match the rest of the script naming convention (sorry if I confused by only fixing some of the others recently)

Comment (NakulManchanda) [reply scripts/explorer-mvs-validate.sh]: renamed

**Pull request merged**